### ISSUE TITLE
use of batchnorm

### ISSUE THREAD

**Issue by zoucheng1991** [hustvl/SparseInst#12]
i noticed that none batch norm is used in Decoder, is it intentional?

**Comment by wondervictor** [comment]
Hi @zoucheng1991, thanks for your interest in SparseInst.
Exactly, we don't use normalization in the encoder and decoder, with all batch normalization frozen in the backbone. Adding group normalization or batch normalization in both encoder and decoder will bring some improvement. For simplicity and speed, we don't adopt any normalization in the encoder or the decoder.

**Comment by yidimopozhibai** [comment]
请问一下跑自己的数据集有评价指标的程序吗

**Comment by wondervictor** [comment]
Hi @yidimopozhibai，对于自己的数据集，你可能需要自己去实现相应的一个评价程序，不过我比较推荐您将自己的数据集转成COCO数据集的格式，这样便能直接使用COCO数据集的评测，也就是项目默认的一个评测方式。这样从数据读取到模型评测，都可以很方便。

**Comment by wondervictor** [comment]
We believe we have answered your question, and as such I'm closing this issue, but let us know if you have further questions. Thanks for your interest in SparseInst 😊 . If you find it useful in your work or research, could you give us a star 🌟 or help us recommend SparseInst to your friends.
